frontend/components/Slider/index.jsx

```diff
@@ -51,12 +54,14 @@ class Slider extends Component {
   /**
    * Fetches the products on component did mouont.
    */
-  componentDidMount() {
-    const { dispatch } = this.props;
-    dispatch(fetchProductRelations({
-      productId: this.props.productId,
-      type: this.props.type,
-    }));
+  componentDidMount () {
+    const { dispatch } = this.props
+    if (this.props.type !== "property") {
```

Comment:
// TODO: move to some common place
const TYPE_PROPERTY = 'property';

```suggestion
    if (this.props.type !== TYPE_PROPERTY) {
```